### PR TITLE
An attempt to fix feature overrides for campaign module.

### PR DESCRIPTION
**Pull request by angaither**
#### What's this PR do?

Updates `dosomething_campaign` feature to hopefully get rid of the features overrides. 
#### How should this be reviewed?

deploy to staging and see if the feature is no longer overridden 
image
#### Relevant tickets

Refs #6735
#### Checklist
- [x] Tested on staging.

### CODE REVIEW

File: lib/modules/dosomething/dosomething_campaign/dosomething_campaign.features.field_instance.inc [hunk 1]
```diff
@@ -3209,17 +3215,18 @@ The title determines the URL of the page. This means that if you are changing th
     'entity_type' => 'node',
     'field_name' => 'title_field',
     'label' => 'Title',
-    'required' => TRUE,
+    'required' => 1,
```
Comment (weerd) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.features.field_instance.inc:3218]: were `TRUE` and `FALSE` not working or did Features just change these automatically?

**Comment by weerd** [comment]
Excellent punctuation additions!

👍

File: lib/modules/dosomething/dosomething_campaign/dosomething_campaign.features.field_instance.inc (hunk 1 above)
Comment (angaither) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.features.field_instance.inc:3218]: features just did this automagically `¯\_(ツ)_/¯`

**Comment by angaither** [comment]
well this worked mostly, but there's still node group overrides on staging, that I don't see locally. nor do they get fixed with a features update. so fun all around.

**Comment by weerd** [comment]
image

**Comment by angaither** [comment]
@weerd you're tellin me

**Pull request merged**